script/config-center/README.md

```diff
@@ -1,6 +1,18 @@
 # Script usage demo
 ![Since 1.1.0](https://img.shields.io/badge/Since%20-1.1.0-orange.svg?style=flat-square)
 
+## required attributes 
+
+You only need to keep the following configuration from 'config.txt'. For more configuration information, please visit [seata.io](https://seata.io/)
+
+| server                   | client                                 |
+| ------------------------ | -------------------------------------- |
+| store.mode               | service.vgroupMapping.my_test_tx_group |
+| store.db.driverClassName | service.default.grouplist              |
```

Comment:
`service.default.grouplist `  only used in `registry.type=file` mode

Reply:
> `service.default.grouplist ` only used in `registry.type=file` mode

PTAL